sources/Directory.Packages.props

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
+  <PropertyGroup>
+    <RoslynPrivateVersion>4.8.0-7.23558.1</RoslynPrivateVersion>

```

Comment:
Shouldn't be needed if the `<PackageVersion>`version is defined in this file.

Reply:
It is not defined in this file but we could mo this to `Stride.RestoreHelper.csproj` and set it directly there.